### DISCUSSION TITLE
Folder selection empty in Firefox

### DISCUSSION THREAD

**Discussion by nirvdrum** [floccusaddon/floccus#1225] [Q&A] [answered]
I had stopped using Floccus for a while due to high CPU usage. That's been fixed, so I've been looking to use Floccus again. I was able to load it in Edge without any issues, but with Firefox I'm unable to select any folders. The list is empty. Has anyone run into this before?

I tried to debug the issue but the shipped extension doesn't appear to have source maps, which made tracking down lines quite difficult. I tried to build the extension, but that failed as well. I think it's probably because I have Node > 14. Next up is to write a Dockerfile that hopefully matches the software versions the project's README indicates. Is there a better way to debug this extension in Firefox?

**Comment by marcelklehr** [comment, marked as answer]
> with Firefox I'm unable to select any folders. The list is empty.

If the list of folders is empty, it might just take a long time to load.
